### PR TITLE
NG1605 - Searchfield Category Compact

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This pull request will fix compact size height.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise-ng/issues/1605

**Steps necessary to review your pull request (required)**:
- Pull this git repository, build, and run the app
- Go to http://localhost:4000/components/searchfield/test-searchfield-category-compact.html
- See the compact size

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Comment by tmcconechy** [comment]
Screenshot 2024-01-11 at 1 35 45 PM

- we might have to make the one bigger so the X doesnt take up the whole thing or hide the X in that case with the setting
- x could be centered vertically (we can consider making it 16x16 as well for compact)

**Review by tmcconechy — APPROVED** [review]

**Review by jbrcna — CHANGES_REQUESTED** [review]
image
image
image

**Failing in Firefox classic theme**
image

File: app/views/components/searchfield/test-searchfield-category-compact.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<div class="row">
+  <div class="one-half column">
+    <div class="field">
+      <input id="searchfield-1" class="searchfield mp-search-input" />
```
Comment (whernebrink) [inline app/views/components/searchfield/test-searchfield-category-compact.html:4]: Please remove `mp-search-input` on the inputs. This is a class we define and use in the experience designer, would just confuse consumers of this enterprise-ng. :)

File: src/components/searchfield/searchfield.js [hunk 2]
```diff
@@ -1599,6 +1600,7 @@ SearchField.prototype = {
       }
 
       targetWidthProp = `calc(${isAlternate ? '100%' : baseWidth} - ${subtractWidth}px)`;
+      console.log(targetWidthProp);
```
Comment (whernebrink) [inline src/components/searchfield/searchfield.js:1603]: Remove log :)

**Review by janahintal — CHANGES_REQUESTED** [review]
Still failing on my end. 
image
image

**Review by glenlieorillo — CHANGES_REQUESTED** [review]
Misaligned close button in LTR and RTL.

https://github.com/infor-design/enterprise/assets/102151352/abb4f23f-e644-4172-9b46-38d3dd0a5c8a

image

**Review by jbrcna — CHANGES_REQUESTED** [review]
image

**Comment by tjamesallen15** [comment]
can you guys specify what browsers are on the screenshots? it will be helpful on my end. @infor-design/qa

**Comment by glenlieorillo** [comment]
Hello @tjamesallen15, I tested this on Mac Chrome. Thanks!

**Comment by jbrcna** [comment]
> Hello @tjamesallen15, I tested this on Mac Chrome. Thanks!


Mac Firefox

**Comment by tjamesallen15** [comment]
> Still failing on my end. image image

May i know what browser is this?

**Comment by janahintal** [comment]
> > Still failing on my end. image image
> 
> May i know what browser is this?

@tjamesallen15 Chrome, Windows 10.

**Comment by janahintal** [comment]
Hi @tjamesallen15 this has been retested and still failing on my end. 

Environment
Browser: All 
OS: Mac & Windows10

image

**Comment by tjamesallen15** [comment]
Done pls check again @janahintal

**Review by tmcconechy — APPROVED** [review]

**Review by janahintal — COMMENTED** [review]
HI @tjamesallen15, border focus still looks off on my end for all OS and Browsers. 
image
image

**Comment by jbrcna** [comment]
**inconsistent borders**
image
notice that the border below overlaps
image

more so  in chrome
image


 
and misaligned in firefox
image


**Classic theme**
**Safari**
image

**Firefox**
image

**Review by glenlieorillo — APPROVED** [review]
image

image

**Review by janahintal — APPROVED** [review]
image
image
image

**Review by ericangeles — APPROVED** [review]

**Pull request merged**